.github/workflows/ai-nightlies.yml

```diff
@@ -0,0 +1,66 @@
+name: Firebase AI Nightlies
```

Comment:
Aren't nightlies "daily release builds"? as defined in https://gradle.org/nightly/? AFAIK this is different